<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_time</class>
 <widget class="QDialog" name="edit_time">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Time</string>
  </property>
  <property name="windowIcon">
   <iconset theme="remindor-qt">
    <normalon>:/icons/remindor-qt.svg</normalon>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="2" column="3">
    <widget class="QLabel" name="mh_label">
     <property name="text">
      <string>Minute(s)/Hour(s)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QLabel" name="error_label">
     <property name="text">
      <string>From must be before To</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QWidget" name="button_widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_button">
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="icon">
         <iconset theme="gtk-cancel">
          <normalon>:/icons/delete.png</normalon>
         </iconset>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ok_button">
        <property name="text">
         <string>Ok</string>
        </property>
        <property name="icon">
         <iconset theme="dialog-ok">
          <normalon>:/icons/ok.png</normalon>
         </iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QTimeEdit" name="at_time"/>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QTimeEdit" name="to_time">
     <property name="time">
      <time>
       <hour>0</hour>
       <minute>1</minute>
       <second>0</second>
      </time>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QComboBox" name="time_combo">
     <item>
      <property name="text">
       <string>Once</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>every X minutes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>every X hours</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="from_label">
     <property name="text">
      <string>From</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="every_spin">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QTimeEdit" name="from_time"/>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="from_check">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="every_label">
     <property name="text">
      <string>Every</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="at_label">
     <property name="text">
      <string>At</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="time_label">
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="to_label">
     <property name="text">
      <string>To</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>time_combo</tabstop>
  <tabstop>at_time</tabstop>
  <tabstop>every_spin</tabstop>
  <tabstop>from_time</tabstop>
  <tabstop>to_time</tabstop>
  <tabstop>ok_button</tabstop>
  <tabstop>cancel_button</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>from_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>from_time</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>from_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>to_time</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
